src/content/blog/projects.mdx

```diff
@@ -0,0 +1,112 @@
+---
+title: Introducing Projects
+description: Projects are containers for vals, files, and folders to increase the scope of what you can build in Val Town
+pubDate: January 29, 2025
+author: Steve Krouse
+---
+
+import { Image } from "astro:assets";
+import ProjectsExample from "./Projects/Project.png";
+import ValExample from "./Projects/Val.png";
+import DateMe from "./Projects/DateMe.png";
+
+<figure>
+  <Image
+    src={ProjectsExample}
+    alt="Young Street Toronto, ,
+    a blog & newsletter project"
+  />
+  <figcaption>
+    [Young Street Toronto](https://www.val.town/x/vawogbemi/yongeStreetToronto),
+    a blog & newsletter project by [Victor](https://www.val.town/u/vawogbemi)
+  </figcaption>
+</figure>
+
+Today we're releasing **Val Town Projects** in public beta. Projects are containers for vals, files, and folders that are versioned collectively, and deployed on Val Town. Where a Val Town Val is like a GitHub Gist, a Val Town Project is like a GitHub Repo. {/* Dan Levine doesn't like this analogy; maybe remove it? */} Projects dramatically increase the scope of what you can build in Val Town.
+
+## Vals are too small
+
+Projects solve the biggest problem with Val Town: vals are too small. A vals is just a single file. It's incredible what you all have been able to do with them, but they become unwieldy after a thousand lines or so. This pain is especially acute when you're using an LLM that is trying to edit or rewrite hundreds or thousands of lines in a single file.
```

Comment:
I'm thinking something along the lines of:


> You shouldn’t have been able to build an entire personal CRM, or [insert other massive user built val, maybe Nate's?] with a single file. Yet, you did. You chained vals together, hacked around import limitations, and manually stitched together forks and updates to keep things running. Most users either stuffed all their code into a single super long val, just made all their changes directly to their main production vals (sometimes breaking it briefly), or moved off Val Town entirely.


And then we would get rid of the entire next paragraph